apps/challenges/models.py

```diff
@@ -265,7 +265,7 @@ def __init__(self, *args, **kwargs):
         max_length=2128, null=True, blank=True
     )  # Max length of repository name and tag is 2000 and 128 respectively
     allowed_submission_file_types = models.CharField(
-        max_length=200, default=".json, .zip, .txt, .tsv, .gz, .csv, .h5, .npy"
+        max_length=200, default=".json, .zip, .txt, .tsv, .gz, .csv, .h5, .npy, .npz"
```

Comment:
Why are we changing the default value?

Reply:
We can change the database value of this particular challenge, no?

Reply:
Sure, I thought you meant we should add `npz` to default values too

Reply:
No no, the whole point of having a database field is to have different values for different challenges.